tests/apis.js

```diff
@@ -52,80 +51,300 @@ test.cb.after.always(t => {
+    t.is(Object.keys(result).length, 2, 'Should contain two main objects.');
+  })
+  .catch(e => {
+    console.log(e); // eslint-disable-line no-console
+  });
 });
 
 //////////////////////////////
 // Utils - format
 //////////////////////////////
-test('Utils: Format Results - List', t => {
-  const formatted = apiUtils.format(content.slice(0, 9));
-
-  formatted.forEach(item => {
-    t.true(item.hasOwnProperty('id'), 'Contains ID');
-    t.true(item.hasOwnProperty('type'), 'Contains Type');
-    t.true(item.hasOwnProperty('type_slug'), 'Contains Type Slug');
-    t.true(item.hasOwnProperty('key'), 'Contains Key');
-    t.true(item.hasOwnProperty('key_slug'), 'Contains Key Slug');
-    t.true(item.hasOwnProperty('meta'), 'Contains Meta');
-    t.false(item.hasOwnProperty('attributes'), 'Does not contain attributes');
-    t.is(item.meta.url, `/api/types/${item.type_slug}/${item.id}`, 'URL points to full content item');
+test.serial.skip('Utils: Format Results - List', t => {
+  const formatted = apiUtils.format(live.slice(0, 9));
+
+  return formatted.then(result => {
+    result.forEach(item => {
+      t.true(item.hasOwnProperty('id'), 'Contains ID');
+      t.true(item.hasOwnProperty('type'), 'Contains Type');
+      t.true(item.hasOwnProperty('type_slug'), 'Contains Type Slug');
+      t.true(item.hasOwnProperty('key'), 'Contains Key');
+      t.true(item.hasOwnProperty('key_slug'), 'Contains Key Slug');
+      t.true(item.hasOwnProperty('meta'), 'Contains Meta');
+      t.false(item.hasOwnProperty('attributes'), 'Does not contain attributes');
+      t.is(item.meta.url, `/api/types/${item.type_slug}/${item.id}`, 'URL points to full content item');
+    });
   });
 });
 
-test('Utils: Format Results - Attributes', t => {
-  const item = Math.round(Math.random() * (content.length - 1));
-  let expected = cloneDeep(content[item]);
+test.serial.skip('Utils: Format Results - Attributes', t => {
+  const query = {
+    depth: 2,
+  };
+  const item = Math.round(Math.random() * (live.length - 1));
+  let expected = cloneDeep(live[item]);
 
   if (expected === undefined) {
-    expected = cloneDeep(content[(content.length - 1)]);
+    expected = cloneDeep(live[(live.length - 1)]);
   }
 
   const model = allTypes.find(typ => {
     return typ.id === expected['type-slug'];
   });
 
-  const formatted = apiUtils.format([expected], model.attributes);
-
-  formatted.forEach(itm => {
-    t.true(itm.hasOwnProperty('id'), 'Contains ID');
-    t.true(itm.hasOwnProperty('type'), 'Contains Type');
-    t.true(itm.hasOwnProperty('type_slug'), 'Contains Type Slug');
-    t.true(itm.hasOwnProperty('key'), 'Contains Key');
-    t.true(itm.hasOwnProperty('key_slug'), 'Contains Key Slug');
-    t.false(itm.hasOwnProperty('meta'), 'Does not contain Meta');
-    t.true(itm.hasOwnProperty('attributes'), 'Contains attributes');
+  const formatted = apiUtils.format([expected], model.attributes, allTypes, query);
+
+  return formatted.then(result => {
```

Comment:
Going to want to check that references got populated correctly in the `format `function too